cmake/AddInputFileTests.cmake

```diff
@@ -37,7 +37,8 @@ function(add_single_input_file_test INPUT_FILE EXECUTABLE CHECK_TYPE TIMEOUT)
   elseif("${CHECK_TYPE}" STREQUAL "execute")
     add_test(
       NAME "${CTEST_NAME}"
-      COMMAND sh ${PROJECT_BINARY_DIR}/tmp/InputFileExecuteAndClean.sh
+      # This script is written below, and only once
+      COMMAND sh ${PROJECT_BINARY_DIR}/tmp/input_file_execute_and_clean.sh
```

Comment:
Why is this snake_case now?

Reply:
because `generate_and_check_docs.sh` is also snake_case

Reply:
Dang it me... (I probably wrote that script and named it wrong). Let's move to CamelCase like the rest of the code :) I can rename the `generate_and_check_docs.sh` in a separate PR

Reply:
Not your fault but mine ;) It's the tmp script written for the docs postprocessing in `SetupDoxygen.cmake`. I'll fix it

Reply:
Fixed in the last commit